### ISSUE TITLE
functioning registration

### ISSUE THREAD

**Issue by guaka** [guaka/hitchticker#22]
@UserStefan #21  "Creating a functioning registration. Seems like all dudes are posting anonymous. Not sure if it should be obligatory, but it shouldn´t be like this."

**Comment by guaka** [comment]
How should it be? What should be changed?

**Comment by UserStefan** [comment]
The old one was like this and i liked this simple process:
"Send a first SMS with some specific text like "Register Stefan". The ticker automatically registers your number and adds Stefan as your username. All the following messages from your phone will be posted as User Stefan."

**Comment by UserStefan** [comment]
Also this regards to an online registration. The problem i had was, that all people are posting anonymous, which is kinda sad.

**Comment by simison** [comment]
Registration works, they're anonymous probably if the message is coming via SMS. #23

**Comment by UserStefan** [comment]
Shouldn´t be a cellphone number be connected to a username? This is what i am understanding about registration.

**Comment by simison** [comment]
Yeah true, sorry I didn't elaborate enough; that bit indeed isn't implemented. :-)

**Comment by UserStefan** [comment]
Is this working now? Can´t check it without mobile.

When SMS, Registration and Hashtags are ready, we bring our baby in public.

**Comment by guaka** [comment]
Is registration actually broken? If yes, keep this open.
Or it is just the link with SMS that's broken? In that case, close this issue, focus on #23.
